### PR TITLE
`azurerm_cognitive_deployment` - update description of property `capacity`

### PR DESCRIPTION
**Pull request by liuwuliuyun**
Fixes https://github.com/hashicorp/terraform-provider-azurerm/issues/22900

### CODE REVIEW

File: website/docs/r/cognitive_deployment.html.markdown [hunk 1]
```diff
@@ -78,7 +78,7 @@ A `scale` block supports the following:
 
 * `family` - (Optional) If the service has different generations of hardware, for the same SKU, then that can be captured here. Changing this forces a new resource to be created.
 
-* `capacity` - (Optional) Tokens-per-Minute (TPM). If the SKU supports scale out/in then the capacity integer should be included. If scale out/in is not possible for the resource this may be omitted. Default value is `1`. Changing this forces a new resource to be created.
+* `capacity` - (Optional) Tokens-per-Minute (TPM). Unit of this value is thousands of Token-per-Minute. Default value is `1` which indicates limitation is `1000` tokens per minute. If the SKU supports scale out/in then the capacity integer should be included. If scale out/in is not possible for the resource this may be omitted. Changing this forces a new resource to be created.
```
Comment (WodansSon) [inline website/docs/r/cognitive_deployment.html.markdown:81]: That is way better, but is this maybe more clear? Also should we add a link to the official product documentation explaining what this field does exactly? 

```suggestion
* `capacity` - (Optional) Tokens-per-Minute (TPM). The unit of measure for this field is in the thousands of Tokens-per-Minute. Defaults to `1` which means that the limitation is `1000` tokens per minute. If the resources SKU supports scale in/out then the capacity field should be included in the resources configuration. If the scale in/out is not supported by the resources SKU then this field can be safely omitted. Changing this field forces a new resource to be created.
```

**Review by WodansSon — CHANGES_REQUESTED** [review]
@liuwuliuyun, thanks for the PR, I have left a comment with a question. If you could take a look that would be great! 🚀

File: website/docs/r/cognitive_deployment.html.markdown (hunk 1 above)
Comment (liuwuliuyun) [reply website/docs/r/cognitive_deployment.html.markdown:81]: Great, and this is the link for TPM https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/quota?tabs=rest

**Comment by liuwuliuyun** [comment]
Thanks for the comment, I added the link to TPM but not sure the words I choose. Feel free to change it.

File: website/docs/r/cognitive_deployment.html.markdown [hunk 2]
```diff
@@ -78,7 +78,7 @@ A `scale` block supports the following:
 
 * `family` - (Optional) If the service has different generations of hardware, for the same SKU, then that can be captured here. Changing this forces a new resource to be created.
 
-* `capacity` - (Optional) Tokens-per-Minute (TPM). If the SKU supports scale out/in then the capacity integer should be included. If scale out/in is not possible for the resource this may be omitted. Default value is `1`. Changing this forces a new resource to be created.
+* `capacity` - (Optional) Tokens-per-Minute (TPM). The unit of measure for this field is in the thousands of Tokens-per-Minute. Defaults to `1` which means that the limitation is `1000` tokens per minute. If the resources SKU supports scale in/out then the capacity field should be included in the resources' configuration. If the scale in/out is not supported by the resources SKU then this field can be safely omitted. Here is an [Azure doc](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/quota?tabs=rest) explaining TPM in details. Changing this field forces a new resource to be created.
```
Comment (WodansSon) [inline website/docs/r/cognitive_deployment.html.markdown:81]: This looks good, but should reword the link to the docs a little?
```suggestion
* `capacity` - (Optional) Tokens-per-Minute (TPM). The unit of measure for this field is in the thousands of Tokens-per-Minute. Defaults to `1` which means that the limitation is `1000` tokens per minute. If the resources SKU supports scale in/out then the capacity field should be included in the resources' configuration. If the scale in/out is not supported by the resources SKU then this field can be safely omitted. For more information about TPM please see the [product documentation](https://learn.microsoft.com/azure/ai-services/openai/how-to/quota?tabs=rest).
```

**Review by WodansSon — CHANGES_REQUESTED** [review]

**Review by WodansSon — APPROVED** [review]
@liuwuliuyun Thanks for pushing those changes, this LGTM now! 🚀

**Pull request merged**